plugins/mac-virtualcam/src/obs-plugin/cmake/macos/Info.plist.in

```diff
@@ -23,6 +23,8 @@
 	<key>LSMinimumSystemVersion</key>
 	<string>${CMAKE_OSX_DEPLOYMENT_TARGET}</string>
 	<key>NSHumanReadableCopyright</key>
-	<string>(c) 2012-${CURRENT_YEAR} Hugh Bailey</string>
+	<string>(c) 2022-${CURRENT_YEAR} Sebastian Beckmann, Patrick Heyer</string>
```

Comment:
Is this change correct?

Reply:
For context, on an earlier version of this PR on the Info.plist in the `camera-extension` directory I asked for this change (and I assume it then got changed for both files).
On this Info.plist here (in `obs-plugin`) I think neither the old nor the new is technically correct, it should probably be along the lines of `2020-${CURRENT_YEAR} John Boiles, Patrick Heyer, Sebastian Beckmann`.